service/src/main/java/com/commercetools/pspadapter/payone/IntegrationService.java

```diff
@@ -53,6 +56,19 @@ public void start() {
 
         Spark.port(port());
 
+        redirect.any("/", "/health", Redirect.Status.MOVED_PERMANENTLY);
```

Comment:
Why we can not use ```/health``` for testing too? One day you might decide base URL has to do something special. I would not mix service with explicit ```/health``` endpoint as it is not obvious and maybe be confusing for outsiders